mlflow/gateway/fluent.py

```diff
@@ -45,3 +45,21 @@ def search_routes(search_filter: Optional[str] = None) -> List[Route]:
         List[Route]: A list of Route instances representing the found routes.
     """
     return MlflowGatewayClient().search_routes(search_filter)
+
+
+@experimental
+def query(route: str, data):
+    """
+    Issues a query request to a configured service through a named route on the Gateway Server.
+
+    This function creates an instance of MlflowGatewayClient and uses it to issue the query to the
+    provided route with the provided data.
+
+    Args:
```

Comment:
Can we `:param:` syntax?

Reply:
no idea why I reverted to numpy docstring syntax for that file... fixing all of those entries.